app/mailers/authentication_mailer.rb

```diff
@@ -32,6 +33,19 @@ def reset_password_instructions(record, token, opts = {})
     )
   end
 
+  def unlock_instructions(record, token, _opts = {})
+    unlock_link = unlock_url(record, unlock_token: token)
+    template_id = GOV_NOTIFY_CONFIG['unlock_account']['template_id']
+
+    UseCases::Administrator::UnlockAccountEmail.new(
```

Comment:
I would call this `SendUnlockEmail` - I think this can be confused with a use case that unlocks an account.